native/cocos/bindings/auto/jsb_ar_auto.cpp

```diff
@@ -601,6 +601,49 @@ static bool js_cc_ar_ARModule_getCameraDepthBuffer(se::State& s)
 }
 SE_BIND_FUNC(js_cc_ar_ARModule_getCameraDepthBuffer) 
 
+static bool js_cc_ar_ARModule_getTexInitFlag(se::State& s)
+{
+    CC_UNUSED bool ok = true;
+    const auto& args = s.args();
+    size_t argc = args.size();
+    cc::ar::ARModule *arg1 = (cc::ar::ARModule *) NULL ;
+    bool result;
+    
+    if(argc != 0) {
+        SE_REPORT_ERROR("wrong number of arguments: %d, was expecting %d", (int)argc, 0);
+        return false;
+    }
+    arg1 = SE_THIS_OBJECT<cc::ar::ARModule>(s);
+    if (nullptr == arg1) return true;
```

Comment:
Why change this `auto generated` file? This PR doesn't make any change of swig configurations.

Reply:
I just make the action ' node generate.js '. Maybe someone forget to make the action

Reply:
pull the latest code and execute `node genbindings.js` again.

Reply:
I had pull the latest code